src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -1,92 +1,101 @@
 import * as React from "react";
-import bem from "../../utils/bem";
+import {
+  Breadcrumb as ChakraBreadcrumb,
+  BreadcrumbItem,
+  BreadcrumbLink,
+  useStyleConfig,
+} from "@chakra-ui/react";
+import generateUUID from "../../helpers/generateUUID";
+import { ColorVariants } from "./BreadcrumbsTypes";
 import Icon from "../Icons/Icon";
-import { IconNames } from "../Icons/IconTypes";
+import { IconNames, IconRotationTypes, IconSizes } from "../Icons/IconTypes";
 
-type breadcrumb = { url: string; text: string };
+//  * Map the ButtonType to the Button Chakra theme variant object. If a wrong
```

Comment:
Nit: Use the comment block syntax: `/**` to start and `*/` to end.

Reply:
Good catch.  I got caught inbetween.